### PR TITLE
Automate chaosmonkey cron setup

### PR DESCRIPTION
**Pull request by syamantm**
proposed solution for https://github.com/Netflix/chaosmonkey/issues/13

* added new configs
```
[install]
run_cron_expression = ""                                         # main cron expression, default is 2 hours before "start_time"
run_cron_path = "/etc/cron.d/chaosmonkey-schedule"               # path to cron config    
run_script_path = "/apps/chaosmonkey/chaosmonkey-schedule.sh"    # cron script path
log_path = "/var/log"                                            # default log directory for cron job
```

* Use run_cron_expression to install the chaosmonkey cron.
* added a new command `install` which will :
   * Install cron.
   * run database migration.

### CODE REVIEW

File: command/install.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	if err != nil {
+		return err
+	}
+
+	cronExpr, err := cfg.InstalledCronExpression()
+	if err != nil {
+		return err
+	}
+
+	crontab := fmt.Sprintf("%s %s %s", cronExpr, cfg.TermAccount(), cfg.InstalledScriptPath())
```
Comment (lorin) [inline command/install.go]: I'd like to have a newline (`\n`) at the end of the crontab, it makes the file easier to read when you cat it:

**Comment by lorin** [comment]
@syamantm  I was testing this out locally, and one thing I noticed is that if I want to change the locations of the chaosmonkey-terminate and chaosmonkey-schedule scripts from the default, the config file looks like this:

```
[chaosmonkey]
term_path = "/usr/local/bin/chaosmonkey-terminate"

[install]
run_script_path = "/usr/local/bin/chaosmonkey-schedule"

[spinnaker]
endpoint = "http://localhost:8084"

[database]
host = "localhost"
name = "chaosmonkey"
user = "root"
encrypted_password = "password"
```

If we are going to support auto-install, it seems inconsistent to put one of the generated shell scripts in the `[chaosmonkey]` section and  the other in the `[install]` section. 

What do you think?

**Comment by syamantm** [comment]
@lorin that is a very good point. I'm not entirely sure a new `install` section in the config is actually necessary, can't see it's adding much value at this point. can put all the new params under `chaosmonkey`. thoughts ?

**Comment by lorin** [comment]
@syamantm I like the idea of putting these params under `[chaosmonkey]`, let's do that.

**Comment by lorin** [comment]
I would also recommend recommended renaming `run_script_path` to something like `schedule_path`, so it's consistent with `term_path` naming.

**Comment by lorin** [comment]
I think there was some confusion on the proposed name change.
The current implementation looks like this:


```toml
[chaosmonkey]
term_path = "/apps/chaosmonkey/chaosmonkey-terminate.sh"
script_path = "/apps/chaosmonkey/chaosmonkey-schedule.sh"
schedule_path = "/etc/cron.d/chaosmonkey-schedule"
```

What I intended wth my suggested name change was for term_path to point to
the shell script that terminates an instance, and for schedule_path to point to
the shell script that generates the termination schedule.

Let's do it this way:

```toml
[chaosmonkey]
term_path = "/apps/chaosmonkey/chaosmonkey-terminate.sh"
schedule_path = "/apps/chaosmonkey/chaosmonkey-schedule.sh"
schedule_cron_path = "/etc/cron.d/chaosmonkey-schedule"
```

**Comment by syamantm** [comment]
@lorin sorry about the confusion. done.

**Pull request merged**